debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/LogMinerDmlParserTest.java

```diff
@@ -336,4 +336,38 @@ public void shouldParsingRedoSqlWithParenthesisInFunctionArgumentStrings() throw
         assertThat(entry.getOldValues().get(1).getColumnData()).isNull();
         assertThat(entry.getNewValues()).isEmpty();
     }
+
+    @Test
+    @FixFor("DBZ-3413")
+    public void testParsingDoubleSingleQuoteInWhereClause() throws Exception {
```

Comment:
Could we either add tests or adjust this specific test to verify this double single quote behavior in both an `INSERT` statement's column value as well as in an `UPDATE` statement's set-clause for completeness?

Reply:
Sure, I added those test cases into this test.

Reply:
Thanks!